config/templates/crdb-tls-example.yaml.in

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 The Cockroach Authors
```

Comment:
Can we generate the year as well.  One less file to update its header

Reply:
Done.